Content.Client/Power/SMES/SmesVisualizerComponent.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Content.Client.Power.SMES;
+
+[RegisterComponent]
+public sealed class SmesVisualsComponent : Component
+{
```

Comment:
Same as APC, should probably use SmesComponent

Reply:
Moving smescomponent to shared is way more doable in this case than apc component so you should give it a shot